loafwallet/LanguageSelectionViewController.swift

```diff
@@ -0,0 +1,57 @@
+import UIKit
+
+class LanguageSelectionViewController: UITableViewController {
+	let viewModel = LanguageSelectionViewModel()
+
+	var titleLabel: UILabel!
```

Comment:
🥲
Not really into force unwrap....it's hidden poison. 

Why not just :

```
var titleLabel = UILabel()
```

Reply:
👍

Reply:
I agree completely, I hate force unwraping, sry. 😱